pkg/soperator/crdVersionMap.go

```diff
@@ -21,29 +21,17 @@ under the License.
 
 package soperator
 
+import "github.com/blackducksoftware/synopsys-operator/pkg/util"
+
 // SOperatorCRDVersionMap is a global lookup table in the package. It maps versions of the operator
 // to the resource versions it is compatible with
 var SOperatorCRDVersionMap = operatorCRDVersionMap{
 	versionMap: map[string]operatorVersions{
 		"latest": {
```

Comment:
What exactly is this map for and why is only latest needed?

Reply:
It is used to migrate the CRD version from one version to another.. So I thought to move the operator also similar to app structure..